### PR TITLE
v0.2.0 preparations

### PR DESCRIPTION
**Pull request by shibumi**
*Please fill in the fields below to submit a pull request.  The more
information that is provided, the better.*

**Fixes issue #**:

None

**Description of pull request**:

This PR updatest he go.mod/go.sum files and adds a changelog for the upcoming 0.2.0 release.

**Please verify and check that the pull request fulfills the following
requirements**:

- [X] Tests have been added for the bug fix or new feature
- [X] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Comment by shibumi** [comment]
No idea where the test coverage decrease comes from.

**Comment by adityasaky** [comment]
It's rather odd, because #111 dropped coverage to 92.655% which this PR also reports. Not sure what the comparison is with.

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* Go 1.16
+* Go 1.15
+
+Changes:
+* support for ITE-6 data structures
+* support for ITE-5 signature generation
+* support for specifying a local directory when running inspections
+* switch from golang.org/x/crypto/ed25519 to crypto/ed25519
+
+Thanks to our contributors: Aditya Sirish, Christian Rebischke, cindykimxp, Dan Lorenc, Fredrik Skogman, Lukas Puehringer, Omer Levi Hevroni, Radu M, Santiago Torres, Scott Buckel, seb-bah, Vaibhav Lodha
```
Comment (adityasaky) [inline Changelog.md:15]:
```suggestion
Thanks to our contributors: Aditya Sirish, Christian Rebischke, Cindy Kim, Dan Lorenc, Fredrik Skogman, Lukas Puehringer, Omer Levi Hevroni, Radu M, Santiago Torres, Scott Buckel, seb-bah, Vaibhav Lodha
```

Comment (adityasaky) [reply Changelog.md:15]: Not sure who seb-bah is.

Comment (SantiagoTorres) [inline Changelog.md:15]: I like this!

**Comment by SantiagoTorres** [comment]
LGTM!

File: Changelog.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Supported Go versions for this release:
+* Go 1.16
+* Go 1.15
+
+Changes:
+* initial release of in-toto-golang
+* support for in-toto-verify
+* support for in-toto-run
+
+Thanks to our contributors: Aditya Sirish, ayush159, Christian Rebischke, Dan Lorenc, Lukas Puehringer, Santiago Torres, Vaibhav Lodha
```
Comment (adityasaky) [inline Changelog.md:28]:
```suggestion
Thanks to our contributors: Aditya Sirish, Ayush Sethi, Christian Rebischke, Dan Lorenc, Lukas Puehringer, Santiago Torres, Vaibhav Lodha
```

**Review by adityasaky — APPROVED** [review]
LGTM, minor nits!

File: Changelog.md (hunk 1 above)
Comment (shibumi) [reply Changelog.md:15]: it's someone who did not configure his git client properly...

**Pull request merged**